### PR TITLE
fixed lazy expression issue

### LINKED ISSUE #188: lazy regex occurence number
In the [specifications](https://json-schema.org/understanding-json-schema/reference/regular_expressions.html) about regex, in the last four lines : 
```
{x}: Match exactly x occurrences of the preceding regular expression.
{x,y}: Match at least x and at most y occurrences of the preceding regular expression.
{x,}: Match x occurrences or more of the preceding regular expression.
{x}?, {x,y}?, {x,}?: Lazy versions of the above expressions.
```
a lazy version of `{x}` is mentioned. However, this lazy version has no meaning because `{x}` matches **exactly** x occurrences.

### PR DESCRIPTION
**Pull request by Sanket-0510**
**GitHub Issue:** #188 
Fixes #188 

**Summary**:

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

No
its a small fix, which solves the lazy expression issue

### CODE REVIEW

**Comment by benjagm** [comment]
@gregsdennis Can you please help my reviewing if the proposed solution is ok?

**Comment by gregsdennis** [comment]
Please hold. I have a question in the issue. This may not be necessary.

**Review by gregsdennis — APPROVED** [review]
Discussion in issue.

**Review by benjagm — APPROVED** [review]

File: pages/understanding-json-schema/reference/regular_expressions.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ described below.
     preceding regular expression.
 -   `{x,}`: Match `x` occurrences or more of the preceding regular
     expression.
--   `{x}?`, `{x,y}?`, `{x,}?`: Lazy versions of the above expressions.
+-   `{x,y}?`, `{x,}?`: Lazy versions of the above expressions.
```
Comment (karenetheridge) [inline pages/understanding-json-schema/reference/regular_expressions.md:50]:
```suggestion
-   `{x,y}?`, `{x,}?`: Non-greedy versions of the above expressions.
```

While we're here, I'd like to make this fix (it's the only occurrence in the file).

Comment (karenetheridge) [reply pages/understanding-json-schema/reference/regular_expressions.md:50]: Otherwise, your edit is correct.

**Review by karenetheridge — CHANGES_REQUESTED** [review]

**Review by karenetheridge — APPROVED** [review]

**Pull request merged**